src/api/ledger/parse/suspended-payment-finish.js

```diff
@@ -0,0 +1,25 @@
+/* @flow */
+'use strict';
+const assert = require('assert');
+const sjclcodec = require('sjcl-codec');
+const utils = require('./utils');
+
+function convertHexToString(hexString) {
+  const bits = sjclcodec.hex.toBits(hexString);
+  return sjclcodec.utf8String.fromBits(bits);
+}
+
+function parsePayment(tx: Object): Object {
+  assert(tx.TransactionType === 'SuspendedPaymentFinish');
+
+  return utils.removeUndefined({
+    memos: utils.parsePaymentMemos(tx),
+    owner: tx.Owner,
+    offerSequence: tx.OfferSequence,
```

Comment:
We should have rippled 0.29.1 release this month but suspended payments feature will not be enabled. We will enable it in rippled release 0.3 which will probably happen next month.